draft-ietf-jose-hpke-encrypt.md

```diff
@@ -128,26 +128,40 @@ The specification chooses a specific subset of the HPKE features to use with JOS
 Hybrid Public Key Encryption (HPKE) {{I-D.ietf-hpke-hpke}} is a public key encryption
 (PKE) scheme that provides encryption of arbitrary-sized plaintexts given a
 recipient's public key.
+This specification enables JSON Web Encryption (JWE) {{RFC7516}} to leverage HPKE,
+bringing support for HPKE encryption and KEMs to JWE,
+and the possibility of utilizing future HPKE algorithms, including hybrid KEMs.
```

Comment:
I think the term hybrid can be super confusing in this context and think some/many folks are starting to use the word composite a la LAMPS' Composite ML-KEM 
```suggestion
and the possibility of utilizing future HPKE algorithms, including composite KEMs.
```